show/feature.py

```diff
@@ -62,9 +62,10 @@ def feature_status(db, feature_name):
     if feature_name:
         key = "FEATURE|{}".format(feature_name)
         if feature_name in cfg_table:
-            data = cfg_table[feature_name]
+            data = {}
             if keys and (key in keys):
-                data.update(dbconn.get_all(dbconn.STATE_DB, key))
+                data = dbconn.get_all(dbconn.STATE_DB, key)
+            data.update(cfg_table[feature_name])
```

Comment:
Curious!
What demands this change. Read state-db first & later update from CFG.

Reply:
How does this command shows the current state of the feature ?

Reply:
This command shows configured state, same as it did before. The command showing the actual state is not in this PR.